src/components/basic/DlPanelContainer/DlPanelContainer.vue

```diff
@@ -224,7 +224,7 @@ export default defineComponent({
         }
     },
     computed: {
-        isStudioMode(): boolean {
+        isStudioMode() {

```

Comment:
I don't understand why this is reverted, it was causing errors
https://github.com/dataloop-ai/components/commit/4391dc4995a9c95e41dff5fcfe68152207d2de9d